azure-ts-appservice-devops/README.md

```diff
@@ -0,0 +1,88 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# A Todo App on Azure App Service with SQL Database and Application Insights and deploys it to Azure DevOps
+
+A Todo List application from Azure Samples GitHub: [.NET Core MVC sample for Azure App Service](https://github.com/azure-samples/dotnetcore-sqldb-tutorial), a web app built with ASP.NET Core, Entity Framework Core and a SQL database. 
+
+Provisions Azure SQL Database and Azure Application Insights to be used in combination with App Service. Defines an Azure DevOps pipeline to deploy in CI/CD environment.
+
+## Running the App manually with Pulumi CLI
+
+1.  Build and publish the ASP.NET Core project:
+
+    ```
+    $ dotnet publish src
+    ```
+
+1.  Navigate to `infra`:
+
+    ```
+    $ cd infra
+    ```
+
+1.  Create a new stack:
+
+    ```
+    $ pulumi stack init dev
+    ```
+
+1.  Login to Azure CLI (you will be prompted to do this during deployment if you forget this step):
+
+    ```
+    $ az login
+    ```
+
+1.  Restore NPM dependencies:
+
+    ```
+    $ npm install
+    ```
+
+1. Define SQL Server username:
+
+    ```
+    pulumi config set azure-appservice-devops:sqlUsername <value>
+    ```
+
+1. Define SQL Server password (make it complex enough to satisfy Azure policy):
+
+    ```
+    pulumi config set --secret azure-appservice-devops:sqlPassword <value>
+    ```
+
+1.  Run `pulumi up` to preview and deploy changes:
+
+    ``` 
+    $ pulumi up
+    Previewing changes:
+    ...
+
+    Performing changes:
+    ...
+    info: 10 changes performed:
+        + 10 resources created
+    Update duration: 1m14.59910109s
+    ```
+
+1.  Check the deployed website endpoint:
+
+    ```
+    $ pulumi stack output endpoint
+    https://dev-as10d706a2.azurewebsites.net
+    $ curl "$(pulumi stack output endpoint)"
+    <!DOCTYPE html>
+    <html>
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Index - My ToDoList App</title>
+    ...
+    ```
+
+## Integrating with Azure DevOps
+
+`azure-pipeline.yml` in the root folder of this example shows a configuration for Azure DevOps using [Pulumi task](https://marketplace.visualstudio.com/items?itemName=pulumi.build-and-release-task).
```

Comment:
Do you mind adding a note here about the `pulumi.access.token` build variable and that one can get a value for it from https://app.pulumi.com/account/tokens? It isn't clear from just looking at these build configuration files where one might get that. And looking at our pulumi.io site, I can't find a page that shows that either, though, it is referenced by the `pulumi login` command. I'll work with someone on the team to fix that.

Reply:
Added a line to README, please check if you think it's descriptive enough.

Reply:
Looks good to me, @mikhailshilkov. Thank you!